src/app/shared/top-panel/resource-menu/resource-detail/resource-status-line/resource-status-line.component.ts

```diff
@@ -50,6 +51,7 @@ export class ResourceStatusLineComponent implements OnChanges {
 
   public ngOnChanges(changes: SimpleChanges): void {
     if (this.organization) {
+      this.store.dispatch(new UsersAction.Get({organizationId: this.organization.id}));
```

Comment:
Is this really the right component which should be responsible for fetching the data? It seems that the organization is passed to it as an input parameter so why don't we fetch the data there?